src/tedi/components/buttons/collapse/collapse.module.scss

```
@@ -224,8 +224,8 @@
       }
 
       &:focus-visible .tedi-collapse__icon-wrapper {
-        outline-color: var(--button-main-neutral-inverted-border-focus);
         background: var(--button-main-neutral-inverted-icon-only-background-focus);
+        outline-color: var(--button-main-neutral-inverted-border-focus);
```

Stylelint (17.13.0)

[error] Expected "outline-color" to come before "background" (order/properties-order)

(order/properties-order)